components/docs-chef-io/content/automate/ha_config_gen.md

```diff
@@ -129,9 +129,11 @@ In Case of AWS managed or Customer managed databases below fields will be requir
 ## Backup
 
 - If configuring backup at the time of deployment the  we many need following fields:
-  - `Bucket name` for object storage or AWS S3 type of backup provide bucket name, for AWS deployment bucket will be created if not exist in AWS.
+  - `Bucket name` for object storage or Google Cloud Storage bucket name in google cloud platform or AWS S3 type of backup provide bucket name or , for AWS deployment bucket will be created if not exist in AWS.
```

Comment:
```suggestion
  - `bucket_name` for object storage or Google Cloud Storage bucket name in google cloud platform or AWS S3 type of backup provide bucket name or , for AWS deployment bucket will be created if not exist in AWS.
```
The heading introductory line of the section says, "fields". But "Bucket name" is a prompt, and the storage type it shows are "options" Can we modify the tag as it is confusing now?

Reply:
updated the introductory line